src/Symfony/Component/Routing/Loader/AttributeClassLoader.php

```diff
@@ -53,7 +53,7 @@
  */
 abstract class AttributeClassLoader implements LoaderInterface
 {
-    protected string $routeAnnotationClass = RouteAnnotation::class;
+    protected string $routeAnnotationClass = RouteAttribute::class;
```

Comment:
I didn't find usage of this variable and setter `setRouteAnnotationClass` outside this class.

How can we manage this ?

1/ Rename method to `setRouteAttributeClass` with BC layer + deprecated `$routeAnnotationClass` to `routeAttributeClass`
2/ Deprecate method + property without replacement and delete in next major
3/ other ?

Reply:
I'd go with 2/ but I'd make the property private in 8.0, not just renamed - it doesn't need to be protected

Reply:
I'd go with 1/ sorry (rename with deprecation layer)

Reply:
I added deprecations, can you tell me if it's ok ?